File: demo/src/main/java/com/baozi/demo/item/news/NewsItemBean.java
package com.baozi.demo.item.news;

import com.baozi.demo.item.news.NewsImageItem;
import com.baozi.demo.item.news.NewsItem;
import com.baozi.treerecyclerview.annotation.TreeDataType;

/**
 * @author jlanglang  2017/7/5 17:34
 * @版本 2.0
 * @Change
 */
@TreeDataType(iClass = NewsItem.class)
public class NewsItemBean {
    public String title;
    public int images;
}

File: demo/src/main/java/com/baozi/demo/item/news/NewsItemBean.kt
package com.baozi.demo.item.news

import com.baozi.treerecyclerview.annotation.TreeDataType

/**
 * @author jlanglang  2017/7/5 17:34
 * @版本 2.0
 * @Change
 */
@TreeDataType(iClass = NewsItem::class)
class NewsItemBean {
    var title: String? = null
    var images: Int = 0
}
